packages/vue/src/components/organisms/SfSidebar/SfSidebar.vue

```diff
@@ -167,20 +170,11 @@ export default {
       }
     },
     classHandler() {
-      let update = false;
       if (this.staticClass !== this.$vnode.data.staticClass) {
         this.staticClass = this.$vnode.data.staticClass;
-        update = true;
       }
       if (this.className !== this.$vnode.data.class) {
         this.className = this.$vnode.data.class;
-        update = true;
-      }
-      if (update) {
-        this.position =
-          [this.staticClass, this.className].toString().search("--right") > -1
-            ? "right"
-            : "left";
       }
     },
```

Comment:
It can be removed if you add dynamical class changing on props value toggling in line 2.

Reply:
Now we have `position` props,  and animation based on this props. We don't need `classHandler` method and `className`, `staticClass` variables. I removed this code. They were needed to handle the animation in old way like below:

```
if (this.className !== this.$vnode.data.class) {
    this.className = this.$vnode.data.class;
    update = true;
 }
if (update) {
   this.position =
   [this.staticClass, this.className].toString().search("--right") > -1
      ? "right"
       : "left";
}
```